### DISCUSSION TITLE
All links to documentation broken

### DISCUSSION THREAD

**Discussion by ghost** [openssl/openssl#24987] [Q&A]
Due to the website redesign pretty much *all* links to documentation are broken, including everything that search engines indexed. Instead, it all redirects to the (honestly completely useless) home page, without even saying the link is broken.

**Please** add rewrite rules that work, and don't break UX for everyone searching basic information.

**Comment by t-j-h** [comment]
This is being worked on - the website cut over is rather new - and we will be addressing and doc or download redirects that are missing as fast as we can.

**Comment by t-j-h** [comment]
The doc redirects should be in place - you may need to clear any bad redirects from the interim if your browser has cached one of those redirects before we got the correct ones in place.

The intent was always to have the prior doc links remaining working (via redirects).

If you hit one that isn't working can you drop it as an example in here.

**Reply by mattcaswell** [reply]
This is definitely not working. Example:

https://www.openssl.org/docs/man3.1/man3/SSL_set_options.html

Ends up here:

https://docs.openssl.org/master/

**Reply by t-j-h** [reply]
Noted ...

**Reply by t-j-h** [reply]
Many links work but not all of the redirects are in place (as @mattcaswell  has noted) - this is being worked on at the moment.

**Reply by kulkarniamit** [reply]
All links of "New Blog post:" in https://openssl-library.org/news/newslog/index.html are 404 right now.
Example: [QUIC and OpenSSL](https://www.openssl.org/blog/blog/2020/02/17/QUIC-and-OpenSSL/) link in [newslog](https://openssl-library.org/news/newslog/index.html)

**Reply by mattcaswell** [reply]
@vavroch2010

**Comment by tom-cosgrove-arm** [comment]
This seems to be a duplicate of #24984

**Comment by tom-cosgrove-arm** [comment]
Sorry, found another broken link. At https://openssl-library.org/about/committers/index.html it says

> The current list of committers is listed below

and *committers* is a broken link (goes to https://openssl-library.org/policies/omc-bylaws#committers)

**Comment by t-j-h** [comment]
All the old blog pages should now be redirecting correctly ...

**Reply by kulkarniamit** [reply]
Thank you for fixing the redirects. 
A few posts are still 404:
* [04-May-2017	New Blog post: Using TLS1.3 with OpenSSL](https://openssl-library.org/blog/blog/2017/05/04/tlsv1.3/) is 404
* [05-Jun-2020 	New Blog post: OpenSSL 3.0 Alpha3 Release](https://openssl-library.org/blog/blog/2020/06/05/OpenSSL3.0Alpha3/) is 404
* [25-Jun-2020 	New Blog post: OpenSSL 3.0 Alpha4 Release](https://openssl-library.org/blog/blog/2020/06/25/OpenSSL3.0Alpha4/) is 404
* [05-Sep-2020 	New Blog post: OpenSSL Is Looking for a Full Time Administrator and Manager](https://openssl-library.org/blog/blog/2020/09/05/OpenSSL.ProjectAdminRole/) is 404
* [21-Oct-2020 	New Blog post: OpenSSL 3.0 Alpha7 Release](https://openssl-library.org/blog/blog/2020/10/20/OpenSSL3.0Alpha7/) is 404
* [17-Jun-2021 	New Blog post: OpenSSL 3.0 Release Candidate](https://openssl-library.org/blog/blog/2021/06/17/OpenSSL3.0ReleaseCandidate/) is 404
* Images in https://openssl-library.org/post/2016-10-12-f2f-rt-github/ are missing

**Comment by Perimental1** [comment]
Okey, im read

**Comment by tom-cosgrove-arm** [comment]
Sorry, just found another.

The top Google hit for "OpenSSL release strategy" is https://www.openssl.org/policies/releasestrat.html which doesn't redirect to anywhere

**Reply by mhsmith** [reply]
The current page is here: https://openssl-library.org/policies/general/release-policy/

**Reply by t8m** [reply]
This one is fixed now.

**Comment by beldmit** [comment]
https://openssl.org/blog/ also returns 404

**Reply by quarckster** [reply]
openssl.org doesn't have its own blog anymore, blog posts are spread across all OpenSSL web sites.

**Comment by mhsmith** [comment]
And another one: https://www.openssl.org/news/openssl-1.1.1-notes.html

With the final release of OpenSSL 1.1.1 being only a year ago, it seems a bit early to throw it down the memory hole.

**Reply by quarckster** [reply]
This one is fixed.

**Comment by tom-cosgrove-arm** [comment]
Another broken link, this time the top Google hit for "OpenSSL 1.1.1 EOL" which is https://www.openssl.org/blog/blog/2023/03/28/1.1.1-EOL/ redirects to https://openssl-library.org/post/2023-03-28-1.1.1-EOL/ but gives 404

**Reply by tom-cosgrove-arm** [reply]
This is still an issue

**Reply by quarckster** [reply]
This one is fixed. cc @t-j-h

**Reply by tom-cosgrove-arm** [reply]
Hmm, the redirect from the original link https://www.openssl.org/blog/blog/2023/03/28/1.1.1-EOL/ goes to https://openssl-library.org/post/2023-03-28-1.1.1-EOL/ but the article is now at https://openssl-library.org/post/2023-09-11-eol-111/index.html

**Reply by tom-cosgrove-arm** [reply]
Ah - presumably two articles about 1.1.1 EOL - one in March 2023 with a reminder of EOL coming in Sept 2023, and one in Sept 2023 with the final EOL notice. The former is the first result in a Google search for `openssl 1.1.1 eol`, and definitely used to exist: https://web.archive.org/web/20240322014939/https://www.openssl.org/blog/blog/2023/03/28/1.1.1-EOL/

**Reply by quarckster** [reply]
> Hmm, the redirect from the original link https://www.openssl.org/blog/blog/2023/03/28/1.1.1-EOL/ goes to https://openssl-library.org/post/2023-03-28-1.1.1-EOL/ 

No, it redirects to <https://openssl-library.org/post/2023-03-28-1.1.1-eol/>:

```
curl -I https://www.openssl.org/blog/blog/2023/03/28/1.1.1-EOL 
HTTP/2 301 
content-type: text/html
location: https://openssl-library.org/post/2023-03-28-1.1.1-eol/
x-cloud-trace-context: ec92953c858658606a47ab1740c2a927
content-length: 169
date: Thu, 19 Dec 2024 12:33:45 GMT
server: Google Frontend
via: 1.1 google
strict-transport-security: max-age=31536000; includeSubDomains; preload
alt-svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000
```

**Reply by tom-cosgrove-arm** [reply]
Okay, must be local cache in my Chrome instance. I see the difference is in the case of "eol" at the end of the URL. Thanks for the correction.

**Comment by t8m** [comment]
Is @quarckster following this?

**Reply by quarckster** [reply]
Now I am following, thanks for mentioning me.
